src/resampler.rs

```diff
@@ -116,6 +116,15 @@ impl State {
     pub fn get_output_latency(&self) -> usize {
         unsafe { speex_resampler_get_output_latency(self.st) as usize }
     }
+
+    pub fn set_quality(&self, quality: usize) -> usize {
+        unsafe { speex_resampler_set_quality(self.st, quality as i32) as usize}
+    }
+
+    pub fn get_quality(&self, quality: &mut i32) {
```

Comment:
it should return the value instead of writing the mutable reference.

Reply:
yep, we were not sure if you wanted to be completly adererent to the c api

Reply:
The high level API should be as rustic as possible :)

Reply:
patched